modules/gitops-argo-cd-round-robin-sharding-algorithm.adoc

```diff
@@ -0,0 +1,138 @@
+// Module included in the following assemblies:
+//
+// * argocd_instance/argo-cd-cr-component-properties.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-argo-cd-round-robin-sharding-algorithm_{context}"]
+= Enabling the round-robin sharding algorithm
+
+:FeatureName: The `round-robin` sharding algorithm
+include::snippets/technology-preview.adoc[]
+
+By default the Argo CD Application Controller uses the non-uniform `legacy` hash-based sharding algorithm to assign clusters to shards. This can result in eneven cluster distribution. You can enable the `round-robin` sharding algorithm to achieve more equal cluster distribution across all shards. 
```

Comment:
```suggestion
By default the Argo CD Application Controller uses the non-uniform `legacy` hash-based sharding algorithm to assign clusters to shards. This can result in uneven cluster distribution. You can enable the `round-robin` sharding algorithm to achieve more equal cluster distribution across all shards. 
```

Reply:
Oh thank you for noticing this typo!